samples/application-insights/sample.md

```diff
@@ -36,21 +38,25 @@ By default the sample code loads this key from an environment variable called `A
 
 snippet: configure-ai-instrumentation-key
 
+The instrumentation key can be obtained under the Application Insights instance properties pane in the Azure portal.
```

Comment:
I looked in the portal and I have no idea how to find this for instances we have, and I didn't even have to create one. You should point to Azure documentation on how to create one and then also describe exactly the clicks to get to the value you need.